localisation_rus/000_ru_zuwc_united_localisation.csv

```diff
@@ -2886,135 +2886,14 @@ GetHorses_horses;horses;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
+average_lifespan;������� ����������������� �����;;;;;;;;;;;;;
+average_lifespan_percent;������� ����������������� �����;;;;;;;;;;;;;
+text_effect_average_lifespan_immortality;������� ����������������� ����� ������ � �G�����������!;;;;;;;;;;;;;
+AUTOCEPHALOUS_PATRIARCHS;���������� ��������;;;;;;;;;;;;;
+AUTOCEPHALOUS_REL_HEAD_TITLE;�������;;;;;;;;;;;;;
+AUTOCEPHALY_IRO;�Y��������� ���������� ��������!;;;;;;;;;;;;;
+REL_AUTOCEPHALY_DESC;������������� �������� (��� ������� ����� �������� ����������);;;;;;;;;;;;;
+PENTARCHY_DRO;��� ����������� � �������, ������������ $RELIGION$[__], ����� ������ ������������ ��������, ������� �������� �������� ���������. �� ����������� ������ ��������, ������ ������ $REL_HEAD_TITLE$ ����� �������� ����� �� ���������. ������ ���������� ������� ����� �������� �� ������ ������ ��������� ����� ������.;;;;;;;;;;;;;
+AUTOCEPHALOUS_REL_HEAD_TITLE_FEMALE;�������;;;;;;;;;;;;;
+EVTDESC_WCDAC_9993_evil;��� �������� ���� � [From.GetName] ��� ��������� �������� ������������ � ������� � ������� ���������� ��������. ��������� ��������� ��������� ���� �������.;;;;;;;;;;;;;
+EVTDESC_WCDAC_9993_normal;��� �������� ���� � [From.GetName] ��� ��������� ����������� ����������. ������� �������� ���� ���� ����������, ���� ����� �� �����.;;;;;;;;;;;;;
```

Comment:
how am I supposed to translate this? o.O

Reply:
why do you need to translate Russian localization?

Reply:
oh... i just checked al .csv-files... didn't check in which file it was...^^ ignore these comments then